### ISSUE TITLE
The result is biased

### ISSUE THREAD

**Issue by haozhihui110** [yxgeee/FD-GAN#41]
Hello, 
      when reproducing the FD-GAN code, use the command line given in your github, the only difference is batch_size = 16 instead of 256, the final result is only 70.9%, what is the reason for this, it is batch_size The value is too small? Or is there some other reason? 
             Thank you

**Comment by yxgeee** [comment]
Yes, the batch size would affect the final performance, since the training of batch norm layers in the network is critical for optimal performance.
